api/src/main/java/edu/wpi/first/shuffleboard/api/plugin/Plugin.java

```diff
@@ -213,6 +214,13 @@ public void onUnload() {
     return ImmutableList.of();
   }
 
+  /**
+   * Gets the custom JSON serializers that this plugin requires to properly save and load custom components.
+   */
+  public List<ElementTypeAdapter<?>> getSavers() {
```

Comment:
Naming is bad. @JLLeitschuh any recommendations?

Reply:
Converters?